debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/actions/ActionMessages.properties

```diff
@@ -46,6 +46,11 @@ OpenBreakpointMarkerAction_Go_to_File_for_Breakpoint_2=Go to File for Breakpoint
 RelaunchActionDelegate_An_exception_occurred_while_launching_2=An exception occurred while launching
 RelaunchActionDelegate_Launch_Failed_1=Launch Failed
 
+EnableAllBreakpointsAction_0=Enable All Breakpoints
+EnableAllBreakpointsAction_1=Enable All Breakpoints?
+EnableAllBreakpointsAction_2=Enable Breakpoints
```

Comment:
EnableAllBreakpointsAction_2 is not used

Reply:
removed it now